Sources/CodeEditTextView/CodeEditTextView.swift

```diff
@@ -32,7 +32,7 @@ public struct CodeEditTextView: NSViewControllerRepresentable {
         lineHeight: Binding<Double>,
         wrapLines: Binding<Bool>,
         editorOverscroll: Binding<Double> = .constant(0.0),
-        cursorPosition: Published<(Int, Int)>.Publisher? = nil,
+        cursorPosition: Binding<(Int, Int)>,
```

Comment:
Could you add this to the DocC comment above? I don't know why it's not already there.